spring/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: Spring-Way overview
+permalink: /spring-way/
+menuInclude: yes
+menuLink: yes
+menuTopTitle: Spring-Way
```

Comment:
Please move the "spring-way" document to be under the "Guides" menu like all other such topics. As noted in the FAQ [How to create a new document](https://dev.folio.org/faqs/how-to-create-doc/#preparation) "Preparation" section, if it is not a Guidelines document or Tutorial or FAQ, then it belongs under Guides. The space in the top navigation is at its maximum, especially for smaller devices. If every topic added a new menu section, then it would be even more crowded. If we ever do add any more navigation menu sections, then it would instead be "FAQs".

Also, as noted in other PR review notes, there are other OpenAPI alternatives so this option should not have such prominence.

Reply:
Via PR #1096 a new section is available at https://dev.folio.org/guides/#module-development-bases to introduce libraries and frameworks and approaches that form the basis for module development.

Please move the new Spring Way document to `/guides/spring-way/` and adjust the links to it.

Reply:
Thank you. I will do that

Reply:
@dcrossleyau, I fixed it. Please review.